elements/lisk-p2p/src/utils/misc.ts

```diff
@@ -139,8 +143,10 @@ export const getUniquePeersbyIp = (
 	return [...peerMap.values()];
 };
 
-export const constructPeerIdFromPeerInfo = (peerInfo: P2PPeerInfo): string =>
-	`${peerInfo.ipAddress}:${peerInfo.wsPort}`;
+export const constructPeerIdFromPeerInfo = (
```

Comment:
It's not PeerInfo anymore. We should rename the method to `constructPeerId` for example.